BackEnd/BackEndBigChallenge/app/Models/Submission.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Submission extends Model
+{
+    use HasFactory;
+
+    // I think I don't need to specify the third parameter as it is called id
+    // Should I relate this submission to PatientInformation or User?
+
+    // if i choose to put user, i must put user_id as the second parameter and in the third, i
+    // have to specify the column of the Submission i am relating to.
+    public function patient(){
+        return $this->belongsTo(User::class, 'user_id', 'patient_id');
+    }
+
```

Comment:
return $this->belongsTo(User::class) si vas a referenciar el Id del user esta bien

Reply:
Creo que no, porque sino como tercer parámetro quedaría sumbission_id y yo quiero relacionar el id del paciente de la submission